File: 631/res/check/check.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	long double Answer, Output;
	FILE *outf = fopen(secure_getenv("VERDICT"), "r");
	FILE *ansf = fdopen(2, "r");

	fscanf(outf, "%Lf", &Output);
	fscanf(ansf, "%Lf", &Answer);

	long double Ae = abs(Answer - Output);
	long double Re = Ae / Answer;

	if (Ae <= 0.000001 || Re <= 0.000001) {
		return 0;
	} else {
		return -1;
	}
}

File: 631/res/check/check.py
#!/usr/bin/env python3

import sys
import os

with os.fdopen(2) as f:
    Answer = float(f.readline())

try:
    Output = float(sys.stdin.readline())
except:
    exit(-1)

try:
    Ae = abs(Answer - Output)
    Re = Ae / Answer

    if Ae <= 0.000001 or Re <= 0.000001:
        exit(0)
    else:
        exit(-1)
except:
    exit(-1)
